### PR TITLE
Fix php debugger test

### PR DESCRIPTION
**Pull request by tolusha**
### What does this PR do?
Fix tests, minor changes.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/6123

not/required

#### Release Notes

not/required

#### Docs PR

not/required

### CODE REVIEW

File: selenium/che-selenium-test/src/test/resources/projects/plugins/DebuggerPlugin/php-tests/index.php [hunk 1]
```diff
@@ -1,14 +1,4 @@
 <?php include 'lib.php';?>
-/*
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/resources/projects/plugins/DebuggerPlugin/php-tests/index.php]: Why did you remove the license header? We have requested to set license headers above the all our source code in tests including resources.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java [hunk 2]
```diff
@@ -120,7 +120,7 @@ public void shouldDebugCliPhpScriptFromFirstLine() {
     notificationPopup.waitExpectedMessageOnProgressPanelAndClosed("Remote debugger connected");
 
     projectExplorer.openItemByPath(PATH_TO_LIB_PHP);
-    editor.setBreakPointAndWaitActiveState(14);
+    editor.setBreakpoint(14);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java:123]: It's important part of test to ensure that breakpoint becomes an active in some time.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java [hunk 3]
```diff
@@ -168,7 +168,7 @@ public void shouldDebugWebPhpScriptFromNonDefaultPortAndNotFromFirstLine() throw
             NON_DEFAULT_DEBUG_PORT));
 
     projectExplorer.openItemByPath(PATH_TO_LIB_PHP);
-    editor.setBreakPointAndWaitActiveState(14);
+    editor.setBreakpoint(14);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java:173]: It's important part of test to ensure that breakpoint becomes an active in some time.

File: selenium/che-selenium-test/src/test/resources/projects/plugins/DebuggerPlugin/php-tests/lib.php [hunk 4]
```diff
@@ -9,6 +9,8 @@
  * Contributors:
  *   Red Hat, Inc. - initial API and implementation
  */
+
 function sayHello($name) {
-    return "Hello, " + $name;
+    return "Hello, $name";
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/resources/projects/plugins/DebuggerPlugin/php-tests/lib.php:14]: It's interesting to know why did you decide to change the code.

Comment (tolusha) [reply selenium/che-selenium-test/src/test/resources/projects/plugins/DebuggerPlugin/php-tests/lib.php:14]: http://php.net/manual/en/language.operators.string.php

It can be:
```
"Hello, $name"
"Hello, ".$name
```

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java (hunk 2 above)
Comment (tolusha) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java:123]: It becomes active when php debug session starts.
https://github.com/eclipse/che/blob/22f6bb9e0529144bdef81fe057078e1a5d92d0b8/selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java#L127

**Review by vparfonov — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java (hunk 2 above)
Comment (tolusha) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java:123]: Spec doesn't say that breakpoint must be activated immediately. But if breakpoint location is valid debugger can activate it later

File: selenium/che-selenium-test/src/test/resources/projects/plugins/DebuggerPlugin/php-tests/lib.php (hunk 4 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/resources/projects/plugins/DebuggerPlugin/php-tests/lib.php:14]: I see, but you decided to change previous variant from some reasons, didn't it :-)

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java (hunk 2 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java:123]: Test is aimed to ensure that breakpoint has ever activated. We can check it later, but IMHO we should check it anyway to catch an error with activation of breakpoint in future.

Comment (tolusha) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java:123]: Agree, I will check if breakpoint activated after debug sessions is initialized.

File: selenium/che-selenium-test/src/test/resources/projects/plugins/DebuggerPlugin/php-tests/lib.php (hunk 4 above)
Comment (tolusha) [reply selenium/che-selenium-test/src/test/resources/projects/plugins/DebuggerPlugin/php-tests/lib.php:14]: because `+` doesn't concatenate stings in php.

Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/resources/projects/plugins/DebuggerPlugin/php-tests/lib.php:14]: Hm. it explains everything. Good catch!

**Comment by tolusha** [comment]
```
[TEST] Command line: /home/tolusha/projects/che/selenium/che-selenium-test/selenium-tests.sh --test=org.eclipse.che.selenium.debugger.* --threads=2
[TEST]
[TEST] Local results:
[TEST] 	- Run: 	     32
[TEST] 	- Failed:     6
[TEST]
[TEST] REGRESSION (6):
[TEST] 	org.eclipse.che.selenium.debugger.CppProjectDebuggingTest.shouldDebugCppProject
[TEST] 	org.eclipse.che.selenium.debugger.CppProjectDebuggingTest.stopDebug
[TEST] 	org.eclipse.che.selenium.debugger.DebugExternalClassTest.shouldDebugJreClass
[TEST] 	org.eclipse.che.selenium.debugger.InnerClassAndLambdaDebuggingTest.shouldDebugMethodLocalInnerClass
[TEST] 	org.eclipse.che.selenium.debugger.InnerClassAndLambdaDebuggingTest.startDebug
[TEST] 	org.eclipse.che.selenium.debugger.InnerClassAndLambdaDebuggingTest.stopDebug
[TEST]
[TEST]
```

File: selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml [hunk 5]
```diff
@@ -69,25 +69,15 @@
             <class name="org.eclipse.che.selenium.debugger.DebugExternalClassTest">
                 <methods>
                     <exclude name="shouldDebugJreClass"/>
-                    <exclude name="shouldDebugMavenArtifactClassWithSources"/>
-                    <exclude name="shouldHandleDebugOfMavenArtifactWithoutSources"/>
                 </methods>
             </class>
             <class name="org.eclipse.che.selenium.debugger.InnerClassAndLambdaDebuggingTest">
                 <methods>
-                    <exclude name="shouldDebugInnerClass"/>
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml]: *shouldDebugInnerClass* was excluded from the stable test list because of it sometimes fail:
https://ci.codenvycorp.com/view/qa/job/che-integration-tests/58/testReport/org.eclipse.che.selenium.debugger/InnerClassAndLambdaDebuggingTest/shouldDebugInnerClass/

File: selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml [hunk 6]
```diff
@@ -69,25 +69,15 @@
...
-                    <exclude name="shouldDebugLambdaExpressions"/>
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml]: *shouldDebugLambdaExpressions* was excluded from the stable test list because of it sometimes fail: https://ci.codenvycorp.com/view/qa/job/che-integration-tests/58/testReport/org.eclipse.che.selenium.debugger/InnerClassAndLambdaDebuggingTest/shouldDebugLambdaExpressions/

File: selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml [hunk 7]
```diff
@@ -69,25 +69,15 @@
...
                     <exclude name="shouldDebugMethodLocalInnerClass"/>
-                    <exclude name="shouldDebugStaticInnerClass"/>
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml]: *shouldDebugStaticInnerClass* was excluded from the stable test list because of it sometimes fail: https://ci.codenvycorp.com/view/qa/job/che-integration-tests/58/testReport/org.eclipse.che.selenium.debugger/InnerClassAndLambdaDebuggingTest/shouldDebugStaticInnerClass/

File: selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml [hunk 8]
```diff
@@ -69,25 +69,15 @@
...
                 </methods>
             </class>
             <class name="org.eclipse.che.selenium.debugger.MultimoduleProjectDebuggingTest"/>
-            <class name="org.eclipse.che.selenium.debugger.NodeJsDebugTest">
-                <methods>
-                    <exclude name="debugNodeJsTest"/>
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml]: *debugNodeJsTest* was excluded from the stable test list because of it sometimes fail: https://ci.codenvycorp.com/view/qa/job/che-integration-tests/55/testReport/org.eclipse.che.selenium.debugger/NodeJsDebugTest/

File: selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml [hunk 9]
```diff
@@ -69,25 +69,15 @@
             <class name="org.eclipse.che.selenium.debugger.DebugExternalClassTest">
                 <methods>
                     <exclude name="shouldDebugJreClass"/>
-                    <exclude name="shouldDebugMavenArtifactClassWithSources"/>
-                    <exclude name="shouldHandleDebugOfMavenArtifactWithoutSources"/>
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml]: *shouldHandleDebugOfMavenArtifactWithoutSources* was excluded from the stable test list because of it sometimes fail on CI:
https://ci.codenvycorp.com/view/qa/job/che-integration-tests/58/testReport/org.eclipse.che.selenium.debugger/DebugExternalClassTest/shouldHandleDebugOfMavenArtifactWithoutSources/

File: selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml [hunk 10]
```diff
@@ -69,25 +69,15 @@
             <class name="org.eclipse.che.selenium.debugger.DebugExternalClassTest">
                 <methods>
                     <exclude name="shouldDebugJreClass"/>
-                    <exclude name="shouldDebugMavenArtifactClassWithSources"/>
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml]: *shouldDebugMavenArtifactClassWithSources* was excluded from the stable test list because of it sometimes fail on CI:
https://ci.codenvycorp.com/view/qa/job/che-integration-tests/58/testReport/org.eclipse.che.selenium.debugger/DebugExternalClassTest/shouldDebugMavenArtifactClassWithSources/

**Review by dmytro-ndp — APPROVED** [review]
looks good now
